include_core/thrtypes.h

```diff
@@ -158,6 +158,8 @@ typedef struct J9ThreadLibrary {
 	uintptr_t parkSpinCount;
 	uintptr_t parkSleepCount;
 #endif /* defined(OMR_THR_YIELD_ALG) */
+	uintptr_t cpuUtilCache;
+	uintptr_t parkSleepCpuUtilThreshold;
```

Comment:
I don’t see these variables being used outside the scope of the `OMR_THR_YIELD_ALG` flag. Can they be #ifdef’ed under this flag?

```suggestion
	uintptr_t cpuUtilCache;
	uintptr_t parkSleepCpuUtilThreshold;
#endif /* defined(OMR_THR_YIELD_ALG) */
```